tests/integration/utils.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MIT
+
+import requests
+from requests_kerberos import HTTPKerberosAuth
+
+
+class Response:
+    """Contains simplified HTTP response
+
+    :attribute dict data: Data fetched from the Cachito API
+    :attribute int id: ID of the request in Cachito
+    :attribute int status: HTTP status code
+    """
+    def __init__(self, data, id, status):
+        self.data = data
+        self.id = id
+        self.status = status
+
+
+class Client:
+    """Class that simplifies access to the Cachito API
+
+    :attribute string _cachito_api_url: URL of the Cachito API
+    """
+    def __init__(self, cachito_api_url):
+        self._cachito_api_url = cachito_api_url
+
+    def fetch_all_requests(self):
+        """Fetch a list of requests from the Cachito API
+
+        :return: Object that contains response from Cachito API
+        :rtype: Response
+        """
+        resp = requests.get(f"{self._cachito_api_url}requests")
```

Comment:
It might be easier to read if there's a slash before `requests`: `f"{self._cachito_api_url}/requests"`
Consider changing how the Cachito API URL is defined to adjust for this.

Reply:
changed